purchases_flutter_ui/analysis_options.yaml

```diff
@@ -0,0 +1,4 @@
+include: package:flutter_lints/flutter.yaml
```

Comment:
we have some things specified in this file in the root

Reply:
Good point! I think I will just create a symlink for this, since the ignored files in the root shouldn't matter for this one.